resources/content/schema/base/snapshot.json

```diff
@@ -4,10 +4,6 @@
             "required": true,
             "nullable": false
         },
-        "managedSnapshotUUID": {
```

Comment:
Because it would be always identical to "snapshot.uuid" now. I've add ability to specific snapshot_uuid when creating snapshot using volmgr.